metacat-common-server/src/main/java/com/netflix/metacat/common/server/events/MetacatEventBus.java

```diff
@@ -71,7 +76,7 @@ public void postAsync(final ApplicationEvent event) {
      */
     public void postSync(final ApplicationEvent event) {
         log.debug("Received request to post an event {} synchronously", event);
-        CounterWrapper.incrementCounter("metacat.events.sync");
+        registry.counter(LogConstants.CounterEventSync.name()).increment();
```

Comment:
The spectator itself has the counter reference cached. The counter only works in active mode and introducing a local id variable will not save. The gauge is the only one that has passive mode, where caches the reference at application will be useful.